### PR TITLE
Feature/ios page 175

### PR DESCRIPTION
**Pull request by ckoutsiaris**
https://github.wdf.sap.corp/gd/FioriExperience/issues/175

1. Added panels for the tab options
2. Added config for the panels
3. Changed tabs field in frontmatter
4. Css changes
5. See All button

Screenshot_2019-03-12 SAP Fiori

### CODE REVIEW

File: src/pages/guideline/ios/1-01-foundation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+tags:
+  - Foundation
+  - SAP Fiori
+version: '1.01'
+tabs:
+  - {label: "Controls", url: "#", description: "A user interface (UI) control is a visual element on a computer screen that helps humans to interact with the underlying software."}
+  - {label: "Views", url: "#", description: "View is a fundamental building block of the Fiori for iOS design language. It is a container for which information can be displayed."}
+  - {label: "Patterns", url: "#", description: "With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise."}
+  - {label: "Floorplans", url: "#", description: "Floorplan is a hub that provides previews of larger bodies of information sourced from different parts of the app."}
+leftnavorder:
```
Comment (devinea) [inline src/pages/guideline/ios/1-01-foundation.md]: delete leftnavorder object

File: src/pages/guideline/ios/1-01-foundation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (devinea) [inline src/pages/guideline/ios/1-01-foundation.md]: Why is this file moved from guideline_ios -> guideline/ios ?

File: src/pages/guideline_ios/1-01-foundation.md [hunk 3]
```diff
@@ -4,17 +4,21 @@ title: Foundation
...
-  - All
-  - Controls
-  - Views
-  - Patterns
-  - Floorplans
+  - {label: "Controls", url: "#", description: "A user interface (UI) control is a visual element on a computer screen that helps humans to interact with the underlying software."}
+  - {label: "Views", url: "#", description: "View is a fundamental building block of the Fiori for iOS design language. It is a container for which information can be displayed."}
+  - {label: "Patterns", url: "#", description: "With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise."}
+  - {label: "Floorplans", url: "#", description: "Floorplan is a hub that provides previews of larger bodies of information sourced from different parts of the app."}
+leftnavorder:
```
Comment (devinea) [inline src/pages/guideline_ios/1-01-foundation.md]: remove `leftnavorder`

File: src/pages/guideline_ios/1-01-foundation.md [hunk 4]
```diff
@@ -4,17 +4,21 @@ title: Foundation
 date: 2019-01-11T00:15:32.894Z
 featuredImage: "/img/control-button.png"
 description: >-
-  SAP Fiori provides a consistent and holistic user experience for SAP software. By creating visually pleasing designs with a strong focus on ease of use, the experience is intuitive and simple, across all devices. With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise.   
+  SAP Fiori provides a consistent and holistic user experience for SAP software. By creating visually pleasing designs with a strong focus on ease of use, the experience is intuitive and simple, across all devices. With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise.
```
Comment (devinea) [inline src/pages/guideline_ios/1-01-foundation.md]: This file should be used and `src/pages/guideline/ios/1-01-foundation.md` deleted

File: src/pages/guideline/ios/1-01-foundation.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+title: Foundation
+date: 2019-01-11T00:15:32.894Z
+featuredImage: "/img/control-button.png"
+description: >-
+  SAP Fiori provides a consistent and holistic user experience for SAP software. By creating visually pleasing designs with a strong focus on ease of use, the experience is intuitive and simple, across all devices. With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise.
+tags:
+  - Foundation
+  - SAP Fiori
+version: '1.01'
+tabs:
```
Comment (devinea) [inline src/pages/guideline/ios/1-01-foundation.md]: tabs needs to be defined in https://github.com/devinea/mvx19-cms/blob/master/static/admin/config.yml#L77 
so that CMS can edit it. 
However we don;'t want to maintain the description in every page. 
I think we should have a field here like 
category: "Controls"

in config.yml
      - {label: "Category", name: "category", widget: "select", options: ["Controls", "Views", "Patterns","Floorplans"]}

File: src/data/concepts.json [hunk 6]
```diff
@@ -0,0 +1,192 @@
+[{
+    "name": "iOS",
```
Comment (devinea) [inline src/data/concepts.json]: This JSON is not need IMO. 
The image should come from the featuredImage in the graphQL query on the markdown pages. 
Title is also maintained on the markdown.
URL is slug of the page presumably so also in the markdown.

File: src/pages/guideline/ios/index.js [hunk 7]
```diff
@@ -138,6 +161,24 @@ export default class GuidelineIosIndexPage extends React.Component {
 
 export const pageQuery = graphql`
   query {
+    panels: allConceptsJson(filter: { name: { eq: "iOS" } }) {
```
Comment (devinea) [inline src/pages/guideline/ios/index.js]: Can be removed with json

File: src/components/Panel/Panel.js [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+                    width: 260,
+                    height: 180,
+                    backgroundColor: colors.gray_100,
+                    marginBottom: 20,
+                    borderRadius: 6,
+                    display: 'flex',
+                    alignItems: 'center'
+                }}
+            >
+                <img css={{
```
Comment (devinea) [inline src/components/Panel/Panel.js]: Should use gatsby-image ` to load png files.

**Comment by ckoutsiaris** [comment]
Yay, thanks
